load_testing/locustfile.py

```diff
@@ -25,42 +26,133 @@
     {"fields[stop]": "id", "filter[location_type]": "0", "filter[route_type]": "3"},
 ).json()["data"]))
 
+all_routes: list[dict] = requests.get(
+    "https://api-v3.mbta.com/routes",
+    {},
+).json()["data"]
+
+@events.init_command_line_parser.add_listener
+def _(parser):
+    parser.add_argument("--api-key", type=str, env_var="V3_API_KEY", default="", help="API Key for the V3 API. Set to avoid rate limiting.")
+
 class MobileAppUser(HttpUser, PhoenixChannelUser):
-    wait_time = between(15, 20)
+    wait_time = between(5, 20)
     socket_path = "/socket"
 
-    prob_reset_map_data = 0.02
-    prob_reset_location = 0.3
+    prob_reset_initial_load = 0.02
     prob_reset_nearby_stops = 0.3
+    prob_filtered_stop_details = 0.76
 
     location: dict | None = None
+    stop_id: str | None = None
     nearby_stop_ids: list[str] | None = None
-    stops_channel: PhoenixChannel | None = None
-    has_map_data = False
+    alerts_channel: PhoenixChannel | None = None
+    predictions_channel: PhoenixChannel | None = None
+    vehicles_channel: PhoenixChannel | None = None
+    did_initial_load = False
+
+    headers: dict = {}
```

Comment:
Should this be `v3_api_headers` or something? I was confused by the idea of passing an `x-api-key` header to the backend itself, which this doesn't actually do.